src/test/unit_tests/jest_tests/test_nc_upgrade_manager.test.js

```diff
@@ -352,11 +352,11 @@ describe('nc upgrade manager - upgrade config directory', () => {
         const nc_upgrade_manager_custom_dir = new NCUpgradeManager(config_fs, { custom_upgrade_scripts_dir });
 
         beforeEach(async () => {
-            await fs_utils.create_path(custom_upgrade_scripts_dir_version_path, 777);
+            await fs_utils.create_path(custom_upgrade_scripts_dir_version_path, 0o777);
```

Comment:
To avoid such bugs, maybe we will have a constant for the modes and reuse it? (not necessarily in this PR).

Reply:
sure, changed it to config.BASE_MODE_DIR